docs/gettingstarted/demo.md

```diff
@@ -0,0 +1,23 @@
+How to run Kitodo.Production in demo mode
+=================================================================================
+
+Prerequisites:
+--------------
+* Make sure you have [Java SE Development Kit](http://www.oracle.com/technetwork/java/javase/downloads/index.html) in at least version 8 installed
+* Make sure you have [Git](https://git-scm.com/downloads) installed
+* Make sure you have [Maven](https://maven.apache.org/download.cgi) installed
+
+Get the source code
+-------------------
+If you have not done yet, [fork](https://help.github.com/articles/fork-a-repo/) the [project repository](https://github.com/kitodo/kitodo-production) on GitHub. Use Git (using a tool like [TortoiseGit](https://tortoisegit.org/), or the command line) to clone the repository as a subdirectory into your Eclipse workspace.
```

Comment:
Why should I need Eclipse to run this demo application? Every other Java IDE should do the same thing.